toolkit/tools/pkg/imagecustomizerlib/customizeselinux_test.go

```diff
@@ -210,20 +212,77 @@ func verifyKernelCommandLine(t *testing.T, imageConnection *imageconnection.Imag
 	notExistsArgs []string,
 ) {
 	grubCfgFilePath := filepath.Join(imageConnection.Chroot().RootDir(), "/boot/grub2/grub.cfg")
+
+	// Check if grub.cfg exists (non-UKI image)
 	grubCfgContents, err := file.Read(grubCfgFilePath)
-	assert.NoError(t, err, "read grub.cfg file")
+	if err != nil {
```

Comment:
Don’t dynamically check things in test verification logic. The test knows if the image is UKI or not.